src/js/footer/liveChatMinimizeOnReload/index.js

```diff
@@ -0,0 +1,13 @@
+function clickElementWithAriaLabel(label) {
+  const element = document.querySelector(`[aria-label="${label}"]`);
+
+  if (element) {
+    element.click();
+  }
+}
+
+window.addEventListener("load", function () {
+  clickElementWithAriaLabel("Minimize window");
+});
+
+clickElementWithAriaLabel("Minimize window");
```

Comment:
Or this is not needed...

Reply:
yes we should remove this , 
because on refresh this is needed